frontend/src/reducers/cartReducers.js

```diff
@@ -0,0 +1,28 @@
+import { CART_ADD_ITEM, CART_REMOVE_ITEM } from "../constants/cartConstants";
+
+export const cartReducer = (state = { cartItems: [] }, action) => {
+    switch (action.type) {
+        case CART_ADD_ITEM:
+            const item = action.payload
```

Comment:
Codacy has a fix for the issue: [Replace `············const·item·=·action.payload` with `······const·item·=·action.payload;`](https://app.codacy.com/gh/Durotemy/e-commerce/pullRequest?prid=9481957)
```suggestion
      const item = action.payload;
```